setup.cfg

```diff
@@ -0,0 +1,24 @@
+[metadata]
+name = Flask-Multipass-SAML-Groups
+version = 0.0.1
+license = Apache-2.0
+author = Christopher Bartz
```

Comment:
I'm not sure about it but at least for Indico plugins in the Indico charm, author and autor_email are:
author = launchpad.net/~canonical-is-devops
author_email = is-devops-team@canonical.com

Reply:
I've done the same mistake with my PR for the indico plugin. So I can confirm that we should use those

Reply:
I see, hard to tell, I found one author as "Jedi Masters" 🤣 : https://github.com/search?q=org%3Acanonical++path%3A*setup.cfg+author&type=code

In that case we might consider changing it in Indico.

Reply:
It is perfectly fine for me to use `launchpad.net/~canonical-is-devops` as the author. To be safe, is there a policy/agreement on this @mthaddon , or can I choose?

Reply:
I think we should go with the original suggestion of:

```
author = launchpad.net/~canonical-is-devops
author_email = is-devops-team@canonical.com
```

If there are other cases where we have inconsistencies we should address those there too. Having individuals here when the project is owned/maintained by the team leads to problems in the long term.